internal/wire/wire.go

```diff
@@ -60,6 +60,12 @@ func (gen GenerateResult) Commit() error {
 	return ioutil.WriteFile(gen.OutputPath, gen.Content, 0666)
 }
 
+// GenerateOptions holds options for Generate.
+type GenerateOptions struct {
```

Comment:
A different way of doing this would be to prepend the header to the `Content`, maybe as an argument to `Commit`.

Reply:
I considered that, but it doesn't work for the `diff` case, as it never calls `Commit`.